content/4-component-composition/4-slot-fallback/angular/app.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+
+@Component({
+	selector: 'app-root',
+	template: '<app-funny-button>Click me !</app-funny-button>',
```

Comment:
In slot fallback, we want to show the component using the fallback, and the same component not using the fallback: https://component-party.dev/#slot-fallback
The component using the fallback is missing

Reply:
Nice catch, indeed I missed that one, I'm fixing it right away !